src/onegov/feriennet/models/activity.py

```diff
@@ -138,6 +138,10 @@ def undecided(self):
 
         return self.activity.state == 'proposed'
 
+    @property
+    def ticket_deletable(self):
+        return self.activity.state != 'archived' and super().ticket_deletable()
```

Comment:
Reversing the order of these conditions might be slightly more efficient, since I'm not sure if `self.activity` is eagerly loaded or not.